src/modules/naomi/naomiAntiphish.ts

```diff
@@ -18,7 +18,19 @@ export const naomiAntiphish = async (Becca: BeccaLyria, message: Message) => {
     const result = await axios.post<{ message: string }>(
       "https://bad-domains.walshy.dev/report",
       { domain: link }
```

Comment:
```suggestion
    const walshyResult = await axios.post<{ message: string }>(
      "https://bad-domains.walshy.dev/report",
      { domain: link }
    );

```
Separate these into two calls.